packages/attribute-slicer-powerbi/scripts/compile.js

```diff
@@ -1,12 +1,13 @@
 const exec = require('child_process').exec;
 const { spawn } = require('child_process');
 // const isWin = /^win/.test(process.platform);
-const baseExec = `${__dirname}/../node_modules/typescript/bin/tsc`;
+const baseExec = require.resolve("typescript") + `/../../bin/tsc`;
 const tsc = spawn("node", [baseExec]);
 
 let hasErrors = false;
 function checkAndLog(data) {
     data = (data || "") + "";
+    // console.log(data);
```

Comment:
might as well kill this commented out line

Reply:
Good catch